pki/denylist.go

```diff
@@ -219,6 +219,7 @@ func (b *denylistImpl) Subscribe(f func()) {
 // download retrieves and parses the denylist
 func (b *denylistImpl) download() ([]byte, error) {
 	// Make an HTTP GET request for the denylist URL
+	// We do not use our safe http client here since we're downloading from our own resource
```

Comment:
download is from a configurable endpoint. (hidden config flag though)
Probably doesn't matter since it is a trusted endpoint.